src/Frost/Effects/Github.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Frost.Effects.Github where
+
+import Polysemy
+import Polysemy.Error
+import FrostError
+import qualified GitHub.Endpoints.Issues as GHI
+import qualified GitHub.Data.Options as GHO
+import qualified GitHub.Data.Issues as GHDI
+import qualified GitHub.Internal.Prelude as GIP
+import PolysemyContrib
+import qualified Data.Vector as DV
+import qualified Data.List.Split as DLS
+
+type Issue = String
+
+data Github m a where
+  Issues :: String -> Github m [Issue]
+
+makeSem ''Github
+
+runGithubPure :: [Issue] -> Sem (Github ': r) a -> Sem r a
+runGithubPure issues = interpret $ \case
+  Issues _ -> return issues
+
+runGithubIO :: (Member (Embed IO) r, Member (Error FrostError) r) => Sem (Github ': r) a  -> Sem r a
+runGithubIO = interpret $ \case
+  Issues repo -> do
+    (username, reponame) <- either throw (\b -> return b) (parseRepo repo)
```

Comment:
Change ` (username, reponame) <- either throw (\b -> return b) (parseRepo repo)` to 
` (username, reponame) <- either throw return (parseRepo repo)`